### DISCUSSION TITLE
Superdebug implementation?

### DISCUSSION THREAD

**Discussion by jramiroz98** [svecosystem/formsnap#104] [Q&A] [answered]
Hello! Wanted to know if Superdebug works with Formsnap?
I did a few tests and it didn't seem to work. I implemented it just as if I was using Superforms, maybe that's where I am going wrong.

Thanks for your time and for sharing this great project!

**Comment by huntabyte** [comment, marked as answer]
Yes, just set the `debug` prop to `true` on `<Form.Root />`.

**Reply by jramiroz98** [reply]
Thanks a lot! And Merry Christmas!
